LinqToRest.Test/QueryProviderTest.cs

```diff
@@ -139,6 +151,33 @@ private static HttpClient MockHttpClient()
                 new EntityWithQueryableMember("Test2", CreateQuery<EntityWithSimpleMembers>("entitywithqueryablemembers/Test2/")),
             });
 
+                

```

Comment:
Newlines!

Reply:
Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/27/commits/d2ca814d94c0c701fd8dc5dd7d9e32e4274521ae